src/primitives/block.cpp

```diff
@@ -31,3 +31,12 @@ std::string CBlock::ToString() const
     }
     return s.str();
 }
+
+int64_t GetBlockCost(const CBlock& block)
+{
+    // This implements the cost = (stripped_size * 4) + witness_size formula,
```

Comment:
nit: This comment would also be useful in GetTransactionCost, since the reasoning appears identical.